### ISSUE TITLE
Unable to use the password after some reboot

### ISSUE THREAD

**Issue by GlitchedCod** [andro2157/DiscordTokenProtector#58]
**Describe the bug**
When my computer start, I have put a shortcut to DiscordTokenProtector into shell:startup (so that it automatically launch at boot). The window open normally but when I type the password I always got the error "invalid password" when actually it's definitely the right one because when I quit the software and start it again it just works normally.

**To Reproduce**
1. Restart the PC
2. Wait that DiscordTokenProtector start
3. Try to put the password

**Expected behavior**
Password valid, can start Discord as usual.

**Screenshots**
![2022-09-22_21h40_45](https://user-images.githubusercontent.com/44437316/191950934-94f5b47b-18f2-40df-a494-25eb1e5de269.png)

**Setup (please complete the following information):**
- Windows 10 21H2 19044.1645
- Discord Token Protector Version dev-10
- Antivirus Windows Defender

**Logs:**
in the log, the error happened at 21:40
[logging.log](https://github.com/andro2157/DiscordTokenProtector/files/9633161/logging.log)

Thanks for any help !

**Comment by andro2157** [comment]
Hello,

That is a very weird bug, have you tried using the "Start with Windows" option built into DiscordTokenProtector? The problem might be due to an incorrect working directory in your shortcut in shell:startup.

**Comment by GlitchedCod** [comment]
> The problem might be due to an incorrect working directory in your shortcut in shell:startup.

Yeah, I think it's that. Sorry for this question it work now (or maybe the problem will appear again but I think that was the issue).
Since I'm here does it's possible in the futur update to allow opening the window again if I start it from the taskbar ?
Thanks for your solution !

**Comment by andro2157** [comment]
I don't really understand the feature, you can already reopen the window by clicking on the DTP icon in the taskbar ?

**Comment by GlitchedCod** [comment]
I mean from just the icon pinned on the bar and not on the toolbar, I only use it to start Discord so I have just unpin Discord and when I want to use-it I click on the DiscordTokenProtector on my bar directly.
![image](https://user-images.githubusercontent.com/44437316/195864804-45276025-d71e-4bac-bc30-385c5e850353.png)

**Comment by jhhk999** [comment]
just restard DTP

**Comment by variablenine** [comment]
i also get this bug quite often

**Comment by GlitchedCod** [comment]
> just restard DTP

That's a solution but not a fix. 🙁
